bitnami/grafana-loki/templates/index-gateway/servicemonitor.yaml

```diff
@@ -12,6 +12,9 @@ metadata:
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/part-of: grafana-loki
     app.kubernetes.io/component: index-gateway
+    {{- if .Values.metrics.serviceMonitor.labels }}
+    {{- .Values.metrics.serviceMonitor.labels | toYaml | nindent 4 }}
+    {{- end }}
```

Comment:
I checks the `.Values.metrics.serviceMonitor.labels` configuration is not in used. 

this value defined  certainly only use in `ServiceMonitor` resources labels.

If I use the prometheus-operator `serviceMonitorSelector` disovery the  `ServiceMonitor` resource, it will be needs this labels.

I explain this bug in  #28768

Reply:
Please see [this comment](https://github.com/bitnami/charts/pull/28857#discussion_r1728589047)